docs/resources/confluent_connect_artifact.md

```diff
@@ -36,7 +36,7 @@ The following arguments are supported:
 * `display_name` - (Required String) The unique name of the Connect Artifact per cloud, environment scope.
 * `cloud` - (Required String) Cloud provider where the Connect Artifact archive is uploaded.
 * `environment` - (Required Block) The Environment that the Connect Artifact belongs to, for example, `env-abc123`. It supports the following:
-  * `id` - (Required String) The ID of the Environment that the Connect Artifact belongs to.
+* `id` - (Required String) The ID of the Environment that the Connect Artifact belongs to.
```

Comment:
nit: we may want to have some whitespaces here based on https://github.com/confluentinc/terraform-provider-confluent/blob/master/docs/resources/confluent_kafka_cluster.md